src/Models/FileLink.php

```diff
@@ -55,4 +55,12 @@ public function getDefaultTitle(): string
 
         return (string) $this->getDescription();
     }
+
+    /**
+     * Create menu title for File link type
```

Comment:
```suggestion
     * The title that will be displayed in the dropdown for selecting the link type to create.
```
Use this same text for all except `Link` itself.

Reply:
Updated